server/events.go

```diff
@@ -60,6 +60,7 @@ const (
 	serverDirectReqSubj       = "$SYS.REQ.SERVER.%s.%s"
 	serverPingReqSubj         = "$SYS.REQ.SERVER.PING.%s"
 	serverStatsPingReqSubj    = "$SYS.REQ.SERVER.PING"             // use $SYS.REQ.SERVER.PING.STATSZ instead
+	serverReloadReqSubj       = "$SYS.REQ.SERVER.%s.RELOAD"        // with server ID
```

Comment:
Should we have one that will trigger all servers? We use this pattern a bunch where one subject is for all servers and then you can target specific ones as well.

Reply:
1/5 Reloading configs on all servers at the same time may not be great for the SLA? Maybe introduce this as one server at a time and add a general Reload later if requested?